DEVELOPMENT.md

```diff
@@ -476,15 +489,86 @@ which contains a QA checklist. Before publishing packages for a wider audience
 someone from the team goes through the checklist and manually tests the app,
 afterwards the team can evaluate whether the release is up to our standards.
 
-**Title:** `QA: v0.0.14 macOS`\
+**Title:** `QA: v0.0.11 macOS`\
 **Body** [QA.md][qa]
 
-**Title:** `QA: v0.0.14 Linux`\
+**Title:** `QA: v0.0.11 Linux`\
```

Comment:
What's with the version change?

Reply:
To make it consistent with the rest of the instructions.